src/main.rs

```diff
@@ -49,7 +51,13 @@ fn calculate_safety_status(
     has_carrot: bool,
     friends_nearby: i32,
 ) -> bool {
-    todo!()
+    if (!wolves_nearby && day_time) || has_carrot {
+        true
+    } else if (!wolves_nearby && day_time) || (friends_nearby > 3) {
+        true
+    } else {
+        false
+    }
```

Comment:
This is pretty good! But you can actually simplify this quite a lot by using boolean logic all on one line. It might look something like `a || (b && c)`.

Reply:
I did it in one line but if I run `cargo fmt` it expands it.